### PR TITLE
Astro 2.2 blog post

### PR DESCRIPTION
**Pull request by matthewp**
- Direct link: https://deploy-preview-646--astro-www-2.netlify.app/blog/astro-220/

### CODE REVIEW

File: src/content/blog/astro-220.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-220/blog-hero-2_2-release.jpg"
+socialImage: "/src/content/blog/_images/astro-220/og-image-2_2-release.jpg"
+lang: "en"
+---
+
+Astro 2.2 is out! Included in this release are:
+
+- **[CDN support for assets](#cdn-support)**: You can now deploy your assets to a CDN external to your site, and Astro will loads from there.
```
Comment (Yan-Thomas) [inline src/content/blog/astro-220.mdx:14]: "loads" doesn't sound right here, I'd change this sentence to explicitly refer back to "assets".
```suggestion
- **[CDN support for assets](#cdn-support)**: You can now deploy your assets to a CDN external to your site, and Astro will load them from there.
```

File: src/content/blog/astro-220.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- **[image() schema helper API change](#image-schema-api-change)**: (experimental) The image() helper has been moved to be inline with the schema definition.
+
+Additionally Astro 2.2 includes **8 patch changes** and was released alongside **8 integrations** with patch releases.
```
Comment (Yan-Thomas) [inline src/content/blog/astro-220.mdx:17]: Missing comma
```suggestion
Additionally, Astro 2.2 includes **8 patch changes** and was released alongside **8 integrations** with patch releases.
```

File: src/content/blog/astro-220.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+			image: image().refine((img) => img.width >= 200, {
+				message: "image too small",
+			}),
+		}),
+})
+```
+
+Previously `image()` was imported directly from `'astro:content'`. This created bugs where Astro was not able to correctly resolve relative image assets defined in the frontmatter.
+
+Providing the helper as a callback aligns with how Zod suggests doing contextual references. This new callback based schema assignment is currently only available behind the assets flag. If you are not using the image() helper you can continue to assign the schemas directly to the `schema` property on the collection.
```
Comment (Yan-Thomas) [inline src/content/blog/astro-220.mdx:62]: Compound modifiers with 'based" most of the time are hyphenated, I believe that's the case here ("callback based")
```suggestion
Providing the helper as a callback aligns with how Zod suggests doing contextual references. This new callback-based schema assignment is currently only available behind the assets flag. If you are not using the image() helper you can continue to assign the schemas directly to the `schema` property on the collection.
```

**Review by Yan-Thomas — COMMENTED** [review]
Blog post looks great @matthewp! Added a few nit/typo suggestions 🙌

File: src/content/blog/astro-220.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import { defineConfig } from "astro/config"
+
+export default defineConfig({
+	build: {
+		assetsPrefix: "https://cdn.example.com",
+	},
+})
+```
+
+After you've run the build (`astro build`) your HTML (both in SSG and SSR modes) will include links that include the prefix for any assets; scripts and stylesheets.
```
Comment (sarah11918) [inline src/content/blog/astro-220.mdx:35]:
```suggestion
The links for any assets, scripts, and stylesheets after running `astro build` will be prefixed with the configured URL. This applies to both SSG and SSR modes.
```
This is the only sentence I really did switch up. You don't have to use this, so switch it up as necessary! But there were a few fixes to be made to the original, and I think we could get it a bit smoother by changing it around a bit.

File: src/content/blog/astro-220.mdx (hunk 3 above)
Comment (sarah11918) [inline src/content/blog/astro-220.mdx]:
```suggestion
Previously, `image()` was imported directly from `'astro:content'`. However, this created bugs where Astro was not able to correctly resolve relative image assets defined in the frontmatter. Providing the helper as a callback aligns with how Zod suggests doing contextual references. 

This new callback based schema assignment is currently only available behind the `--assets` flag. If you are not using the `image()` helper, you can continue to assign the schemas directly to the `schema` property on the collection.
```
I feel like this splits the paragraphs up better: first paragraph for "mistakes were made" and then second paragraph starting here doesn't hide that this only affects the experimental branch, and provides guidance for what that means to the user.

File: src/content/blog/astro-220.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+})
+```
+
+Previously `image()` was imported directly from `'astro:content'`. This created bugs where Astro was not able to correctly resolve relative image assets defined in the frontmatter.
+
+Providing the helper as a callback aligns with how Zod suggests doing contextual references. This new callback based schema assignment is currently only available behind the assets flag. If you are not using the image() helper you can continue to assign the schemas directly to the `schema` property on the collection.
+
+## Try Astro 2.2
+
+Thank you to everyone who contributed to this outstanding release. In addition to these new features, there were also a bunch of bug fixes throughout the core packages and integrations. See the [complete release notes](https://github.com/withastro/astro/blob/29e7b82b8de0709cbbfaa2583b05f3c10978dda3/packages/astro/CHANGELOG.md#220) to learn more.
```
Comment (sarah11918) [inline src/content/blog/astro-220.mdx:66]:
```suggestion
Thank you to everyone who contributed to this outstanding release. In addition to these new features, there were also several bug fixes throughout the core packages and integrations. See the [complete release notes](https://github.com/withastro/astro/blob/29e7b82b8de0709cbbfaa2583b05f3c10978dda3/packages/astro/CHANGELOG.md#220) to learn more.
```
:sweat_smile:

**Review by sarah11918 — COMMENTED** [review]
Let's see how many things @Yan-Thomas already caught by the time I submit this!  :sweat_smile:

**Review by bholmesdev — APPROVED** [review]
2 minor 2 furious

**Pull request merged**